<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>描述性统计生成器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>保留小数点后几位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>date</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>参数设定</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>偏度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_1"/>
      </item>
      <item row="0" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>中位数</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>标准差</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>总和</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>PP检验</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>JB检验量</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>均值</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>KPSS检验</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>众数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>方差</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>最大值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>最小值</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>ADF检验</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>峰度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>样本数</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
      <item row="2" column="7">
       <widget class="QLineEdit" name="lineEdit_9"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_10"/>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="lineEdit_11"/>
      </item>
      <item row="3" column="7">
       <widget class="QLineEdit" name="lineEdit_12"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_13"/>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="lineEdit_14"/>
      </item>
      <item row="4" column="7">
       <widget class="QLineEdit" name="lineEdit_15"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>运行情况说明</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>导入数据</string>
   </property>
   <property name="toolTip">
    <string>导入数据</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>开始运行</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>初始化</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
